# Request a positive affirmation
---
# Response
string affirmation
time timestamp
